File: jewels-and-stones.kt
class Solution {
    fun numJewelsInStones(J: String, S: String): Int {
      var occurrences: Int = 0
      for (x in 0..J.length - 1) {
        for (y in 0..S.length - 1) {
          if (J[x] == S[y]) occurrences += 1
        }
      }
      return occurrences
    }
}

File: jewels-and-stones.rs
impl Solution {
    pub fn num_jewels_in_stones(jewels: String, stones: String) -> i32 {
        let j: Vec<char> = jewels.chars().collect();
        let v: Vec<char> = stones.chars().collect();
        let mut c = 0;

        for n in j {
            for o in &v {
                if n == *o {
                    c += 1;
                }
            }
        }

        return c;
    }
}
